style.css

```diff
@@ -483,29 +475,281 @@ header {
   border-radius: 10px;
 }
 
-@media only screen and (max-width: 700px) {
-  .form-container {
-    flex-wrap: wrap;
+@media only screen and (min-width: 993px) {
```

Comment:
Great job implementing the changes requested by the earlier reviewer. However, there is always room for improvement.
**OPTIONAL**
It'd be nicer if you managed another media query for the break of elements in the medium-sized screen. Also, it seems your mobile version having an overflow from the beginning. So, it'd be great if you worked on that too.
Good luck and happy coding with the next.